compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -53,7 +53,7 @@ pub(crate) fn finalize(cx: &CodegenCx<'_, '_>) {
         None => return,
     };
 
-    let mut covfun_records = instances_used
+    let mut covfun_records = { instances_used }
```

Comment:
What are these braces for?

Reply:
This is to ensure `instances_used` is consumed here rather than borrowed. `instances_used` borrows from `cx`, but it gets reborrowed below.

Reply:
In that case, I would prefer to just `drop(instances_used)` on a subsequent line.

That seems like a much clearer and less magical way to explicitly drop something.

Reply:
Done